tsconfig.json

```diff
@@ -21,5 +21,5 @@
 			"@woocommerce/*": [ "../packages/*/src" ]
 		}
 	},
-	"exclude": [ "node_modules", "build" ]
+	"exclude": [ "node_modules", "build", "build-module", "dist", "vendor" ]
```

Comment:
Also a little bug fix to tsconfig here, the linter gets overwhelmed by too many included dirs at the moment.